src/inference/src/ie_core.cpp

```diff
@@ -1560,8 +1560,15 @@ CompiledModel Core::compile_model(const std::shared_ptr<const ov::Model>& model,
 CompiledModel Core::compile_model(const std::shared_ptr<const ov::Model>& model,
                                   const std::string& deviceName,
                                   const ConfigMap& config) {
+    std::string name = deviceName;
+    auto new_config = config;
+    auto it = config.find("MYRIAD_DISABLE_MX_BOOT");
+    if (it != config.end() && it->second == "YES") {
+        name = "CPU";
+        new_config.erase("MYRIAD_DISABLE_MX_BOOT");
+    }
     OV_CORE_CALL_STATEMENT({
-        auto exec = _impl->LoadNetwork(toCNN(model), deviceName, config);
+        auto exec = _impl->LoadNetwork(toCNN(model), name, new_config);
```

Comment:
But we need to compile network for myriad in order to get a blob

Reply:
There is one of the ways to solve this. I added a confirmation to compile_tool (in the configure() function).